examples/step-22/doc/intro.dox

```diff
@@ -554,13 +554,13 @@ struct InnerPreconditioner;
 template <>
 struct InnerPreconditioner<2>
 {
-    typedef SparseDirectUMFPACK type;
+  using type = SparseDirectUMFPACK;
```

Comment:
This is funny -- if you had member variables in this class, would the `using` statement now be at the indentation level of the members, or of the `private:` tag?

Reply:
Oh, never mind -- we're in a doxygen file...

Reply:
We get something like
```
 template <>
   struct InnerPreconditioner<2>
   {
   public:
     typedef SparseDirectUMFPACK type;
     int                         j;
     using bla = int;
   };
```

Reply:
@masterleinad -- I had missed initially that this is not in a source file and that consequently the indentation of the original code may have been in error. I meant the initial number of spaces at the top of the line.

Either way, it's all good.